### PR TITLE
PrivateLink Template Inclusion

### PR DESCRIPTION
**Pull request by jpanke**
## Purpose of this pull request

This pull request (PR) is to add a new version of the privatelink template for our requestors to add additional clarity to the process of enablement workflow.

## Affected DevDocs pages

https://devdocs.magento.com/cloud/project/privatelink-service.html#enablement-workflow

-  ...

File was uploaded to the root of where the pages are.  I hope this is acceptable.

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@jpanke thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: src/cloud/project/privatelink-service.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Enabling PrivateLink can take up to 5 business days. Providing incomplete, or in
 
 ### Enablement workflow
 
-The following workflow outlines the enablement process for PrivateLink integration with {{site.data.var.ece}}.
+The following workflow outlines the enablement process for PrivateLink integration with {{site.data.var.ece}}.  Before you submit your support ticket, please fill out the following excel file with the necessary information. [PrivateLink Information Template] (Excel file you can download, has sample data)
```
Comment (BarnyShergold) [inline src/cloud/project/privatelink-service.md:67]:
```suggestion
The following workflow outlines the enablement process for PrivateLink integration with {{site.data.var.ece}}. Before you submit your support ticket, please fill out the following Excel file with the necessary information. [PrivateLink Information Template] (The Excel file, which you can download, has sample data)
```

Is it correct to do this as an XLS sheet? Two issues - 1. Not everyone uses Excel or can read it 2. People might be concerned about it carrying a virus - what's the stance on these two?

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by meker12** [comment]
@jeff-matthews, @jpanke Typically, we don't host download files in the DevDocs repo.  It's preferable for content to be hosted on a shared Cloud storage location.  The easiest would be if the Cloud operations team had a place to host the file that DevDocs can link to.  That way, the operations team can update the file as needed.  We do have an AWS store that we use for some files, but every time you update the file, it requires a DevDocs ticket.

File: src/cloud/project/privatelink-service.md (hunk 1 above)
Comment (jeff-matthews) [reply src/cloud/project/privatelink-service.md:67]: Good questions @barny. We do not store these type of binary files in our repo. We recently removed several large binaries that were taking up space in our repo and added them to a `downloads/` directory on the site. 

Two things come to mind @jpanke:

- Hosting this file somewhere other than devdocs.magento.com where the internal owners can manage changes would be ideal. That way, we can simply link to it from devdocs and any future changes will be available on devdocs as soon as owners update the linked doc.
- If hosting elsewhere is not possible, then the file should be converted to PDF that contains a "last modified date" somewhere in the document itself (not the filename) and uploaded to the `downloads/` directory by someone on the `devdocs-admins` team. The source file must be maintained by the owner on internal systems.

**Comment by jeff-matthews** [comment]
Yes, @meker12 said it better than I did. Thanks @meker.

File: src/cloud/project/privatelink-service.md (hunk 1 above)
Comment (barny) [reply src/cloud/project/privatelink-service.md:67]: I think you meant that comment for @BarnyShergold

Comment (jeff-matthews) [reply src/cloud/project/privatelink-service.md:67]: I did. Sorry about that.

Comment (BarnyShergold) [reply src/cloud/project/privatelink-service.md:67]: S'okay - I still get the email 😄

**Comment by jpanke** [comment]
Hey, wanted to comment here.   We don't have a place to host the file readily available.  We only have this form right now but are working on a ZD form that will replace the excel file once it's in place.  We don't expect to have that done for another few months so we need to update our existing one.  It was suggested by Director Shane Herman to put it on DevDocs with the very instructions instead of relying manually providing the form. 

when we get the ZD form in place we will remove this form and update the instructions.

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
@jpanke, please coordinate with the Support team to host this file on the [Magento Help Center](https://support.magento.com/hc/en-us). Since this Excel file appears to be a support asset, the Help Center is a more appropriate location for it.

We'd be happy to link to it from devdocs once it's uploaded.

**Comment by BarnyShergold** [comment]
@jeff-matthews - Author has not made any feedback on this PR. Should we perhaps close this?

**Comment by jpanke** [comment]
Due to the failure to get the file hosted somewhere, we will look for alternatives and I will make another update to this page later.  We are working on getting a new ZD for published instead.

**Comment by jeff-matthews** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@jeff-matthews the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8239`.

**Pull request closed without merging**